include/rotary_embedding_api.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include "compile_util.h"
+#include "dtype.h"
+#include <cmath>
+#include <cstring>
+#include <iostream>
+
+namespace xft {
+void xftRotaryEmbeddingKernel(DataType dt,
```

Comment:
Why not just name `RotaryEmbeddingKernel` and used as `xft::RotaryEmbeddingKernel`?

Reply:
To keep the api interface consistent, align with this PR，layernorm
: https://github.com/intel/xFasterTransformer/pull/31

Reply:
In that PR, changqing just use `invokeLayerNorm` in namespace xft instead of `xftinvokeLayerNorm`.  Did I miss something?

Reply:
Because we want to maintain the consistency of the API interface to facilitate the users , such as 'Datatype' paramter.  RotaryEmbedding added 'Datatype' parameter,  and  'DataType' declare in in the namespace ‘xft ’，Thus RotaryEmbedding is also under the ‘xft’ namespace. This allows users to maintain a uniform approach when using Layernorm and RotaryEmbedding API